src/paths/library/get-search-all-libraries.yaml

```
@@ -103,6 +95,14 @@
                                     - $ref: "../../models/media-container/search-result/directory/tag-key.yaml"
                                     - $ref: "../../models/media-container/search-result/directory/thumb.yaml"
                                     - $ref: "../../models/media-container/search-result/directory/count.yaml"
+                                      required:
+                                      - key
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 40 but found 38

(indentation)